cmd/start.go

```diff
@@ -70,11 +74,33 @@ $ %s start demo-path2 --max-tx-size 10`, appName, appName)),
 				return err
 			}
 
+			thresholdTime := viper.GetDuration(flagThresholdTime)
+
+			eg := new(errgroup.Group)
+			eg.Go(func() error {
```

Comment:
I think we should be subscribe to incoming blocks and update the clients every time we receive one. I think that's actually better than using a user defined threshold time. Alternatively, relayers could also update the clients every N blocks, with the default value being 1

Reply:
we already submit an update msg for each relay msg. This functionality is primarily intended to prevent the expiry of a client so I think using a threshold time is sufficient and reduces complexity. Updating every block would be wasteful both for gas an on-chain storage